examples/android/MODULE.bazel

```diff
@@ -8,12 +8,7 @@ bazel_dep(name = "bazel_skylib", version = "1.2.1")
 bazel_dep(name = "rules_robolectric", version = "4.10.3", repo_name = "robolectric")
 bazel_dep(name = "rules_java", version = "6.4.0")
 bazel_dep(name = "platforms", version = "0.0.7")
-bazel_dep(name = "rules_kotlin")
```

Comment:
Is it possible to build the examples without using the Bazel integration test path? If so then I'd maybe consider keeping the `local_path_override` so that people don't step into the examples to validate their changes only to realize that it's using the previous stable release of `rules_kotlin` rather than `HEAD`.

Reply:
Not without using the `release_archive` repository rule... which is getting tricky with the iteration they are doing on the repository namespacing. On the other hand, running `bazel test //examples` works just fine to validate changes.